src/compiler/types.ts

```diff
@@ -4808,7 +4808,7 @@ namespace ts {
         ES2018 = 5,
         ES2019 = 6,
         ES2020 = 7,
-        ESNext = 8,
+        ESNext = 99,

```

Comment:
Given that the value for ESNext has _already_ been volatile, I see no reason not to make this one last churn and then have all these values be _not_ volatile. But I would wait for another reviewer just in case I’m not thinking of some crazy edge case.

(CI will tell you this, but does this affect the compiler API baselines?)

Reply:
I think we should do this for ModuleKind.EsNext as while at it

Reply:
Agree, added